src/Type.cpp

```diff
@@ -194,7 +194,7 @@ NonVoidNonVolatileTypeFilter::get_type()
 class ChooseRandomTypeFilter : public Filter
 {
 public:
-	ChooseRandomTypeFilter(bool for_field_var);
+	ChooseRandomTypeFilter(bool for_field_var = false, bool struct_has_assign_ops = false);
```

Comment:
Can we not use the default argument for ```for_field_var```? In the subsequent uses of ```ChooseRandomTypeFilter```, we don't actually rely on this default argument, i.e., we explicitly pass the first argument ```true``` and ```false```when we construct ```ChooseRandomTypeFilter``` objects. I am a little cautious of using default arguments.